crates/egui_kittest/src/snapshot.rs

```diff
@@ -49,6 +137,15 @@ impl SnapshotOptions {
         self.output_path = output_path.into();
         self
     }
+
+    /// Change the number of pixels that can differ before the snapshot is considered a failure.
+    /// Preferably, you should use `threshold` to control the sensitivity of the image comparison.
+    /// As a last resort, you can use this to allow a certain number of pixels to differ.
+    #[inline]
+    pub fn diff_threshold(mut self, diff_threshold: impl Into<i32>) -> Self {
```

Comment:
Since we now have two thresholds, I suggest we use very clear names for them.

One is the threshold per-pixel, e.g. how much a pixel may differ to be considered a failure.
The other is a count of how many pixels are allowed to be above the first threshold (at least if I understand the PR correctly).

So lets call this something like `failed_pixel_count_threshold`. It's a mouthful, but it is also a last resort.